concordium-consensus/src/Concordium/ProtocolUpdate/P1.hs

```diff
@@ -17,14 +17,15 @@ import Concordium.Types.Updates
 import Concordium.GlobalState.BlockState
 import Concordium.Kontrol
 import qualified Concordium.ProtocolUpdate.P1.Reboot as Reboot
+import qualified Concordium.ProtocolUpdate.P1.Memo as Memo
```

Comment:
Since the `Memo` constructor has been renamed to `ProtocolP2`, I think the module should also be renamed to match.